docs/usage.md

```diff
@@ -89,6 +89,7 @@ Sample files that can be used as illustrative references can be found in this re
 | activity_name | str | Activity name |
 | lesson_page | str | Lesson page name |
 | content_id | UUID | Content page UUID |
+| visible | str | Visibility of activity to students |
```

Comment:
Oh, I think we had a miscommunication here. I did not intend for us to change the schema in the outputs. If we decide to that in the future, we can spin up a separate issue for that. Let's back this out. For now I just wanted the input data fixture to include the `visible` column to better reflect reality.